src/main/java/model/User.java

```diff
@@ -6,6 +6,15 @@
     private String email;
     private String password;
     private String role;
+    private Code code;
```

Comment:
if we use code in the order we should not use it here

Reply:
yes, i forget to delete